.gitattributes

```diff
@@ -0,0 +1,8 @@
+/tests                export-ignore
+/.gitattributes       export-ignore
+/.gitignore           export-ignore
+/.travis.yml          export-ignore
+/CHANGELOG.md         export-ignore
+/README.md            export-ignore
```

Comment:
I agree with excluding README and similar files, if you need info you can check online